### PR TITLE
fix(model): pad image tokens using an out-of-vocab index

### PR DESCRIPTION
**Pull request by rebel-eunji**
### 🚀 Summary of Changes

The benchmark random dataset uses pad_token_id (0) as a valid token ID, which conflicts with the assumption that pad_token_id is not used as input. Previously, pad_token_id was only used for padding image tokens.
So this PR adopted a new token, out-of-vocab index to pad image tokens and Gemma 3 works correctly with the benchmark code.

---

### 📌 Related Issues / Tickets

* Resolves #
* Related to #

---

### ✅ Type of Change

* [ ] ✨ Feature (`feature`)
* [x] 🧠 Model support (`model`)
* [ ] 🧬 Core engine changes (`core`)
* [x] 🛠 Bug fix (`bug-fix`)
* [ ] ⚙️ Performance improvement (`perf`)
* [ ] 🔁 Refactor or code cleanup (`refactor`)
* [ ] 📄 Documentation (`docs`)
* [ ] ❓ Other (`other`): please describe

---

### 🧪 How to Test

1. Run `...`
2. Verify output: `...`
3. Edge case tested: `...`

---

### 📸 Screenshots / Logs (if applicable)

---

### 📋 Checklist

* [ ] PR title follows Conventional Commits format
* [ ] This PR is linked to an existing issue
* [ ] The test method is described, and the expected result is clearly stated
* [ ] Relevant documentation has been updated (if applicable)

---

### 💬 Notes

---

### CODE REVIEW

File: vllm_rbln/model_executor/models/optimum/gemma3.py [hunk 1]
```diff
@@ -92,9 +95,15 @@ def __init__(
         vllm_config: VllmConfig,
     ) -> None:
         super().__init__(vllm_config=vllm_config)
+        model_path = self.vllm_config.model_config.model
+        tokenizer = AutoTokenizer.from_pretrained(model_path,
```
Comment (rebel-thkim) [inline vllm_rbln/model_executor/models/optimum/gemma3.py]: Wouldn't this be an overhead?

Comment (rebel-eunji) [reply vllm_rbln/model_executor/models/optimum/gemma3.py]: I agree. So I’ll just use a global variable instead of extracting the vocabulary size from the tokenizer.

File: vllm_rbln/model_executor/models/optimum/gemma3.py [hunk 2]
```diff
@@ -36,6 +35,9 @@
 
 logger = init_logger(__name__)
 
+IMG_PAD_TOKEN_ID = -1
+PAD_TOKEN_ID = 0
+
```
Comment (rebel-thkim) [inline vllm_rbln/model_executor/models/optimum/gemma3.py]: This is code that will be reverted someday, right? 
IMG_PAD_TOKEN_ID = -1 should be fine, but I'm not sure if PAD_TOKEN_ID should be hardcoded.
